src/main/scala/io/iohk/ethereum/network/PeerManagerActor.scala

```diff
@@ -92,17 +93,21 @@ class PeerManagerActor(
       val peerAddresses = outgoingPeersAddresses(peers)
 
       val nodesToConnect = nodesInfo
-        .filterNot{ discoveryNodeInfo =>
+        .filterNot { discoveryNodeInfo =>
           val socketAddress = discoveryNodeInfo.node.tcpSocketAddress
           peerAddresses.contains(socketAddress) || isBlacklisted(PeerAddress(socketAddress.getHostString))
         } // not already connected to or blacklisted
         .take(peerConfiguration.maxOutgoingPeers - peerAddresses.size)
 
+      NetworkMetrics.DiscoveredPeersSize.set(nodesInfo.size)
+      NetworkMetrics.BlacklistedPeersSize.set(blacklistedPeers.size)
+      NetworkMetrics.PendingPeersSize.set(nodesToConnect.size)
```

Comment:
Why using nodesToConnect here and not `pendingPeers`?

Reply:
You're right. I will fix that